src/test/create-user-test.ts

```diff
@@ -120,9 +126,15 @@ describe('createUser mutation', () => {
       birthDate: '1995-07-11',
     };
 
+    const expectedError = {
+      code: 400,
+      message: 'Invalid password',
+      additionalInfo: 'Password must be at least 6 characters long.',
+    };
+
     const response = await postQuery(inputData);
 
-    expect(response.data.errors[0].message).to.equal('Password must be at least 6 characters long.');
+    expect(response.data.errors[0]).to.deep.equal(expectedError);
```

Comment:
Nit:

```suggestion
    expect(response.data).to.deep.equal({ data: null, errors: [expectedError] });
```

(Assim vc testa o body inteiro, não apenas o primeiro erro)